vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -9,6 +9,7 @@ def self.ems_inv_to_hashes(inventory)
     def initialize
       @data = {}
       @data_index = {}
+      @registries = Set.new # should be Set?
```

Comment:
> Don't use an additional cache/index, you can reuse @data_index

@simon3z How would I do that? I need Sets to make sure I don't have any duplicates. Also, I don't have any real unique fields on a registry or image that I can use to save them with ```data_index.store_path```